cap_anndata/cap_anndata.py

```diff
@@ -254,6 +262,14 @@ def overwrite(self, fields: List[str] = None, compression: str = "lzf") -> None:
             for key in self.uns.keys_to_remove:
                 del self._file[f"uns/{key}"]
 
+        if "layers" in fields:
+            for key in self.layers.keys():
+                if self.layers[key] is not NotLinkedObject:
+                    dest = f"layers/{key}"
+                    self._write_elem(dest, self.layers[key], compression=compression)
```

Comment:
not sure if that string will work with backed objects. Actually we don't need to provide the ability to overwrite layers as it's backed. Users have a direct control over the matrix. Maybe we just need to add methods to add and delete layers. Not sure.

Reply:
`keys_to_add` property added